manager/apps/aica_manager/models.py

```diff
@@ -216,6 +272,19 @@ def non_suspicious_destination_flows(self, since: int = 86400) -> List[List[Any]
                 "RETURN n"
             )
             all_results.extend(results)
+        ip_strings = [str(x.address) for x in self.ipv6_address.all()]
```

Comment:
The re-use of ip_strings several times could lead to unexpected behavior. Perhaps these variables aren't needed and the list comprehension could just happen in the for statements?